cdc/owner/changefeed.go

```diff
@@ -411,9 +411,15 @@ func (c *changefeed) tick(ctx cdcContext.Context, captures map[model.CaptureID]*
 		newResolvedTs = prevResolvedTs
 		metricsResolvedTs = newResolvedTs
 	}
+
+	// MinTableBarrierTs should never regress
+	if minTableBarrierTs < c.state.Status.MinTableBarrierTs {
```

Comment:
Why might minTableBarrierTs fall back?

Reply:
As mention above, it's calculate from ddlPuller's reolvedTs, when changefeed is just strated, it may regress to `preValue - 1`.